tmt/steps/prepare/ansible.py

```diff
@@ -127,8 +132,10 @@ class PrepareAnsible(tmt.steps.prepare.PreparePlugin[PrepareAnsibleData]):
 
         prepare --how ansible --playbook one.yml --playbook two.yml --extra-args '-vvv'
 
-    Remote playbooks can be referenced as well as local ones, and both
-    kinds can be intermixed:
+    Remote playbooks - provided as URLs starting with ``http://`` or
+    ``https://`` -, local playbooks - optionally starting with a
+    ``file://`` schema - , and playbooks bundled with collections can be
+    referenced as well as local ones, and all kinds can be intermixed:
```

Comment:
It looks like there are some extra dashes:
```suggestion
    Remote playbooks - provided as URLs starting with ``http://`` or
    ``https://``, local playbooks - optionally starting with a
    ``file://`` schema, and playbooks bundled with collections can be
    referenced as well as local ones, and all kinds can be intermixed:
```

Reply:
Addressed in ceb00541